src/app_engine/constants.py

```diff
@@ -42,6 +42,21 @@ class EventType(object):
   # Event signifying that a room enters the state of having exactly
   # two participants.
   ROOM_SIZE_2 = 'room_size_2'
+  ICE_CONNECTION_STATE_CONNECTED = 'ice_connection_state_connected'
+
+
+class AnalyticsField(object):
```

Comment:
I am leaving these as-is because I want the ability to generate enums for JS and I want them to be namespaced there. It seems easier to me, and cleaner, to represent these as classes in Python to make the generation easier and provide some similarity to the JS counterparts. I did move them to the analytics and analytics_page modules as appropriate.